src/python/pants/backend/python/tasks/build_local_python_distributions.py

```diff
@@ -227,7 +227,8 @@ def _create_dist(self, dist_tgt, dist_target_dir, interpreter,
 
     setup_runner = SetupPyRunner.for_bdist_wheel(
       dist_target_dir,
-      is_platform_specific=is_platform_specific)
+      is_platform_specific=is_platform_specific,
+      interpreter=interpreter)
```

Comment:
OK @CMLivingston and @cosmicexplorer - other issues elsewhere aside - this code never stood a fair chance of working since the selected interpreter was not passed.